olp-cpp-sdk-authentication/.clang-format

```diff
@@ -1 +0,0 @@
-BasedOnStyle : Google
```

Comment:
Why did you remove this file?

Reply:
This file was removed since it affects clang-format for me. We already have top-level one.